internal/cmd/binding/get_binding_test.go

```diff
@@ -44,8 +45,6 @@ var _ = Describe("Get binding command test", func() {
 		client.ListBindingsReturns(&types.ServiceBindings{ServiceBindings: []types.ServiceBinding{binding}}, nil)
 		context := &cmd.Context{Output: buffer, Client: client}
 		command = NewGetBindingCmd(context)
-		client.GetInstanceByIDReturnsOnCall(0, &instance1, nil)
```

Comment:
should this be removed?

Reply:
this is moved to line 59, under section : "Describe("Get service binding", func() {"